cookbook/session/index.rst

```diff
@@ -8,4 +8,6 @@ Sessions
     locale_sticky_session
     sessions_directory
     php_bridge
-    avoid_session_start
\ No newline at end of file
+    avoid_session_start
+    /cookbook/configuration/pdo_session_storage
+    /cookbook/configuration/mongodb_session_storage
```

Comment:
The two lines from the configuration section need to be removed. Sorry for the confusion.